core/src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthRequest.java

```diff
@@ -23,6 +23,7 @@
 import org.elasticsearch.action.IndicesRequest;
 import org.elasticsearch.action.support.IndicesOptions;
 import org.elasticsearch.action.support.master.MasterNodeReadRequest;
+import org.elasticsearch.cluster.health.ClusterHealthStatus;
```

Comment:
Actually not. `ClusterHealthStatus` is still needed by `ClusterHealthRequest`, I did not touch this class explicitly. It's just in there now because I created the new class `ClusterHealth` (now as per your suggestion called `ClusterStateHealth`) and moved it together with some related classes to a dedicated package.